src/components/SubmitPage/SubmitCodeCard.tsx

```diff
@@ -41,6 +41,8 @@ export default function SubmitCodeCard({
       'text/python': ['.py'],
       'text/c': ['.c'],
       'text/cpp': ['.cpp'],
```

Comment:
i wonder if it would work by passing them all in this array like `['.cpp', '.cxx', '.cc']`

Reply:
You are right. I was debugging and forgot to undo changes. But resolved now.